pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -61,6 +63,8 @@ type CronWorkflowSpec struct {
 	WorkflowMetadata *metav1.ObjectMeta `json:"workflowMetadata,omitempty" protobuf:"bytes,9,opt,name=workflowMeta"`
 	// StopStrategy defines if the cron workflow will stop being triggered once a certain condition has been reached, involving a number of runs of the workflow
 	StopStrategy *StopStrategy `json:"stopStrategy,omitempty" protobuf:"bytes,10,opt,name=stopStrategy"`
+	// Schedules is a list of schedules to run the Workflow in Cron format
```

Comment:
```suggestion
	// v3.6 and after: Schedules is a list of schedules to run the Workflow in Cron format
```

This should ideally have a version specifier

Reply:
Fixed in #13369 ([this file](https://github.com/argoproj/argo-workflows/pull/13369/files#diff-b05cc1f792ee440442970a2925db373864b61fb061bfa49df7e573ba0b99f9bc))